lib/systems/include/Controllers/LookupLaunchController.h

```diff
@@ -22,9 +22,10 @@ class TorqueControllerLookupLaunch : public virtual BaseLaunchController
     */
```

Comment:
Why's this called LookupLaunchController in this comment? I'm assuming it's been refactored at some point, but if we're renaming things, **now** would be the time to fix the documentation to be up-to-date.

Reply:
im confused by this, this comment is literally in the look up launch controller definition, also did not write this comment so I am not 100% sure on this

Reply:
Whether or not you **wrote** it, you're now **responsible** for it. If you don't know what it means, then ask Ben. Since we have time now (compared to dec/jan/feb), I'd really like to remove old snippets of code from previous refactoring, rather than allowing it to linger.

Reply:
Is there a reason why we have TorqueControllerLookupLaunch as the name of the class, rather than LookupLaunchController (to match the name of the class)?